repository/src/main/java/org/alfresco/repo/lock/LockServiceImpl.java

```diff
@@ -468,16 +476,16 @@ private void persistLockProps(NodeRef nodeRef, LockType lockType, Lifetime lifet
      */
     private Date makeExpiryDate(int timeToExpire)
     {
-        // Set the expiry date

```

Comment:
Just a refactor with static variable usage, to explain why a certain value is infinity